scripts/setup-project.sh

```diff
@@ -2,6 +2,27 @@
 
 set -e
 
+if [ $(uname -o | grep "Cygwin") ]; then
+  echo "Warning: this script does not support Cygwin. Please use WSL or 'git bash'."
```

Comment:
Is it clear why the symlinks don't work under cygwin? It could be that we just don't check for it in make_sym_link

Reply:
e2studio doesn't seem to deal with Cygwin soft links well.
Cygwin doesn't have mklink (at least in a default setup).